apps/passport-server/test/ratelimit.spec.ts

```diff
@@ -219,4 +268,122 @@ describe("generic rate-limiting features", function () {
       }
     }
   );
+
+  step("underlying data model behaves as expected", async function () {
+    const actionId = uuid();
+    const maxActions = 10;
+    const timePeriod = ONE_HOUR_MS;
+
+    // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+    const consumeToken = async () => {
+      // We are always going to consume the same token type
+      return consumeRateLimitToken(
+        db,
+        "TEST",
+        actionId,
+        maxActions,
+        maxActions,
+        timePeriod
+      );
+    };
+
+    for (let i = 1; i <= maxActions; i++) {
+      MockDate.set(Date.now() + 3600);
```

Comment:
What's the intent of this addition?  I think Date.now() is in milliseconds, but 3600ms (3.6 seconds) is a weirdly arbitrary amount of time which suggests you might intend 1 hour (3600 seconds).  A rate of 10 actions per hour would allow for a new token every 6 minutes.

Separate but related, the docs don't make it clear to me whether MockData alters the behavior of Date.now() or only new Date().  Do you intend this to be adding to the existing mock date, or adding to the real current date?

Reply:
Replaced this with a more sensible constant.

I've also added a comment explaining the intended effect of `MockDate.set()` in these calls (and that it affects `Date.now()`).